### PR TITLE
Hide settings page for 'department_head' #23

### PR DESCRIPTION
**Pull request by rapee**

### CODE REVIEW

**Comment by rapee** [comment]
Close issue #23
Close issue #29

File: config/default/locales/en.json [hunk 1]
```diff
@@ -51,4 +57,4 @@
   "team": {
     "saraban": "Saraban Team"
   }
-}
+}
```
Comment (parnurzeal) [inline config/default/locales/en.json]: newline

File: config/default/locales/th.json [hunk 2]
```diff
@@ -51,4 +57,4 @@
   "team": {
     "saraban": "ทีมสารบรรณ"
   }
-}
+}
```
Comment (parnurzeal) [inline config/default/locales/th.json]: newline

File: src/lib/middleware/auth.js [hunk 3]
```diff
@@ -47,54 +47,61 @@ export {
...
+      }
+      next();
+    })
+    .catch(err => {
+      reset_user(req, res);
+      next(err);
+    });
+  };
 }
+
```
Comment (parnurzeal) [inline src/lib/middleware/auth.js:107]: no newline

**Pull request merged**